src/main/java/com/rapid7/client/dcerpc/mssamr/SecurityAccountManagerService.java

```diff
@@ -73,12 +73,12 @@ public UserHandle openUser(DomainHandle domainHandle, int sid) throws IOExceptio
 	return response.getHandle();
     }
 
-    public AliasHandle openAlias(DomainHandle domainHandle, int sid) throws IOException {
+    public AliasHandle openAlias(UserHandle userHandle, int sid) throws IOException {
```

Comment:
This should be the DomainHandle not UserHandle

Reply:
yep!